features/admin.claims.v1/pages/attribute-verification-settings.tsx

```diff
@@ -100,6 +101,14 @@ const AttributeVerificationSettingsFormPage: FunctionComponent<AttributeVerifica
         featureConfig?.attributeDialects?.scopes?.update
     );
 
+    const isMobileNumberVerificationByPrivilegedUsersSupported: boolean =
+        isFeatureEnabled(featureConfig?.attributeDialects,
+            ClaimManagementConstants.FEATURE_DICTIONARY.get("MOBILE_VERIFICATION_BY_PRIVILEGED_USERS"));
+
+    if (!isMobileNumberVerificationByPrivilegedUsersSupported) {
+        HIDDEN_PROPERTIES.push(CONNECTOR_NAMES.ENABLE_MOBILE_NUMBER_VERIFICATION_BY_PRIVILEGED_USERS);
```

Comment:
Just a reminder to update asgardeo deployment configs so that this remains disabled in SaaS.